interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -157,6 +187,7 @@ public void emptyUnary() throws Exception {
 
   @Test(timeout = 10000)
   public void largeUnary() throws Exception {
+    context.attach();
```

Comment:
Is this part of testing `InProcessTransport`? It's a shame this is so invasive and not just for tests that care.